scripts/install2

```diff
@@ -77,6 +81,10 @@ if [ "${#MESH_NAME}" == 0 ]; then
     export MESH_NAME="tomesh"
 fi
 
+if [[ $RPI_REVISION == "OPI" ]]; then
```

Comment:
I Agree, however in this case I think it is smart to separate DETECTION and ACTION. 

One reason is i looked at the flow of the script and no changes are made in the first part of the script. Script currently is split into  detect the revision, ask a few question and THEN make changes

This is why i opted out to start making changes AFTER the last question is asked.